taigaio/taiga-back
# [TAIGA 6] Add initial docker env

![](https://media.giphy.com/media/FaKV1cVKlVRxC/giphy.gif)

Initial files to create official docker images.

To test:
- check the code
- build an image `docker build -f docker/Dockerfile -t taiga-back:latest`
- use repo:taiga-deploy to test the system with the images in the registry
- use repo:taiga-deploy to launch this image in your local computer (change the image in `docker-compose.yml` and `docker-compose-inits.yml`

Then:
- merge, :pray: 
- :+1:
